Makefile

```diff
@@ -119,10 +119,9 @@ test: test_build
 
```

Comment:
Can you explain why these changes were necessary? It is best to make modifications to support fuzzing as optional (for example by protecting it with a macro or IFDEF).

Reply:
I did not make any change here. It was in the original file as well.

Reply:
Ok, it looks like just a path difference anyways.